app/models/allocation_subject.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class AllocationSubject < ApplicationRecord
+  has_many :subject_specialisms
```

Comment:
```suggestion
  has_many :subject_specialisms, inverse_of: :allocation_subject
```

Reply:
Commited in the `1994-add-allocation-subjects-and-specialism-subjects` branch 👍